stripecardscan/src/main/java/com/stripe/android/stripecardscan/scanui/ScanActivity.kt

```diff
@@ -370,7 +365,7 @@ abstract class ScanActivity : AppCompatActivity(), CoroutineScope {
      * The user cannot scan the required object.
      */
     protected open fun userCannotScan() {
-        launch { scanStat.trackResult("user_cannot_scan") }
+        runBlocking { scanStat.trackResult("user_cannot_scan") }
```

Comment:
When does `user_cannot_scan` get set? This event is currently [not being tracked in Scan Stats](https://git.corp.stripe.com/stripe-internal/pay-server/blob/master/api/lib/card_image_verifications/card_image_verifications/scan_stats.rb#L80-L113)

Reply:
(This applies to some of the other events above as well)

Reply:
ahh this should be `user_missing_card`. good catch